aspnetcore/migration/21-to-22.md

```diff
@@ -166,9 +166,13 @@ Visual Studio's **Auto build on browser request** experience doesn't function wi
 
 ## Update logging code
 
-* Move logging provider initialization to `Startup.ConfigureServices`.
+Recommended logging configuration code did not change from 2.1 to 2.2, but some 1.x coding patterns that still worked in 2.1 no longer work in 2.2.
 
-  2.1 example:
+If your app still does logging provider initialization, filtering, and configuration loading in the `Startup` class, move the code to `Program.Main`:
```

Comment:
If your app does logging provider initialization, filtering, and configuration loading in the `Startup` class, move that code to `Program.Main`: